nengo_loihi/tests/test_loihi_api.py

```diff
@@ -22,3 +25,71 @@ def test_overflow_signed(b):
     y, o = overflow_signed(x, bits=b)
     assert np.array_equal(y, z)
     assert np.array_equal(o, q)
+
+
+@pytest.mark.parametrize('offset', (0, 1))
+def test_decay_magnitude(offset, plt, logger):
+    bits = 12
+    # decays = np.arange(1, 2**bits)
```

Comment:
Probably remove this? Guessing it was changed because it took too long?

Reply:
Yeah. I kind of left it in as a reminder, that if you're making big changes to `decay_magnitude`, then you might want to uncomment this and run the longer test. But it's not a test that we need to run all the time. I don't think we have a good way to handle tests like this right now (where we want a shorter and longer version).